src/__tests__/TuxAnimations-test.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+jest.dontMock('../../Animations/Animation');
+jest.dontMock('../../Animations/Fade');
+jest.dontMock('../../Animations/Fly');
+jest.dontMock('../../Animations/Zoom');
+
+describe('Animations', function () {
+  var React, makeAnimation, Idea, Fade, Fly, Zoom, TestUtils, ideaComponent, mocks;
+
+  beforeEach(function () {
+    //Reset animation components and modules before each test
+    React = require('react/addons');
+    makeAnimation = require('../../Animations/Animation');
+    Idea = require('./testComponents');
+    Fade = require('../../Animations/Fade');
+    Fly = require('../../Animations/Fly');
+    Zoom = require('../../Animations/Zoom');
+    TestUtils = React.addons.TestUtils;
+    TestUtils.mockComponent(Idea);
+
+    //Reset mocks before each test
+    mocks = {
+      ideas: [
+        {
+          id: 1,
+          text: 'Save the world.'
+        },
+        {
+          id: 2,
+          text: 'More saving of the world.'
+        }
+      ]
+    };
+  });
+
+  it("applies a class of 'fly' for a fly transition", function () {
+   ideaFlyComponent = TestUtils.renderIntoDocument(
+    <Fly key={mocks.ideas[0].id} >
+      <Idea key={mocks.ideas[0].id} />
+    </Fly>
+    );
+    var ideaFly = TestUtils.findRenderedDOMComponentWithClass(ideaFlyComponent, 'fly');
+    expect(ideaFly.getDOMNode().className).toEqual('fly');
```

Comment:
This expectation is a little bit redundant because we wouldn't have found the component with `findRenderedDOMComponentWithClass` if it did not have the class of `'fly'`.  Can we update this test just to check if `ideaFly` exists?  Same is true of our other tests.